courses/free-courses-en.md

```diff
@@ -284,6 +283,7 @@
 ### Computer Science
 
 * [Berkeley's CS  61A: Taught using SICP](https://archive.org/details/ucberkeley-webcast-PL3E89002AA9B9879E?tab=collection)
+* [CS50's Introduction to Computer Science](https://cs50.harvard.edu/x/) - David J. Malan (Harvard OpenCourseWare and edX)
```

Comment:
Don't invent titles! Use the title given by the resource.

Reply:
@eshellman  The previous title was incorrect. I myself have taken this course and I can prove it
![image](https://user-images.githubusercontent.com/71313814/193795794-56e184de-f640-4a4e-9bb7-07e7c42222b8.png)
This is CS50x is the course's phrase and the actual name is CS50's Introduction to Computer Science
![image](https://user-images.githubusercontent.com/71313814/193796183-68041e31-2f4d-4f65-a0c1-9466177eca95.png)
On [edX](https://www.edx.org/cs50) there are various CS50 courses with their names
![image](https://user-images.githubusercontent.com/71313814/193796821-6c129a37-3104-45d2-9942-01f82a689845.png)
[This](https://www.edx.org/course/introduction-computer-science-harvardx-cs50x) is the main page of the course